scripts/create_releases_pr.sh

```
@@ -23,6 +23,24 @@ function set_release_date() {
   fi
 }
 
+# First day of the current month, used as the exclusive upper bound for the
+# releases plugin filter. docs-sourcer parses date-only strings as midnight
+# UTC, so passing "last day of previous month" misses releases published
+# after 00:00 UTC on that day. Passing "first day of current month" instead
+# captures everything from the previous month regardless of UTC time.
+function set_cutoff_date() {
+  unameOut="$(uname -s)"
+
+  if [ ${unameOut} = "Linux" ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)